test/integration/scripts/execute.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# Copyright 2019 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+workdir=`pwd`
+cd $workdir
+
+curpath=$PWD
+echo $PWD
+
+modulename=$1
+#run the edge_core bin to run the integration
+go build cmd/edge_core.go
+#dynamically append testManager Module before starting integration test.
+sed -i 's/dbTest/dbTest, testManager/g' conf/modules.yaml
+#restart edge_core after appending testManager Module.
+nohup ./edge_core > edge_core.log 2>&1 &
+sleep 15s
+if pgrep edge_core >/dev/null
+then
+    echo "edge_core process is Running"
+else
+    echo "edge_core process is not started"
```

Comment:
we need to return from here if the process failed to start , else all tests will fail  eventually.

Reply:
@rohitsardesai83 , Yes, Handled failed scenario.